src/backend/gporca/server/src/unittest/gpopt/minidump/CDirectDispatchTest.cpp

```diff
@@ -41,6 +41,8 @@ const CHAR *rgszDirectDispatchFileNames[] = {
 	"../data/dxl/minidump/DirectDispatch-IndexScan.mdp",
 	"../data/dxl/minidump/DirectDispatch-DynamicIndexScan.mdp",
 	"../data/dxl/minidump/InsertDirectedDispatchNullValue.mdp",
+	"../data/dxl/minidump/DD_RandDistTable.mdp",
+	"../data/dxl/minidump/DD_RandDistTable_Disjunction.mdp",
```

Comment:
Sorry if it's a little OCD, but prefer to match existing patterns.

```suggestion
	"../data/dxl/minidump/DirectDispatch-RandDistTable.mdp",
	"../data/dxl/minidump/DirectDispatch-RandDistTable-Disjunction.mdp",
```